### PR TITLE
Improve AnimationPlayer warning for imported scenes

### PR DESCRIPTION
**Pull request by nathanfranke**
image

### CODE REVIEW

**Review by Calinou — APPROVED** [review]

File: editor/animation_track_editor.cpp [hunk 1]
```diff
@@ -6112,8 +6112,11 @@ float AnimationTrackEditor::snap_time(float p_value, bool p_relative) {
 }
 
 void AnimationTrackEditor::_show_imported_anim_warning() {
-	// It looks terrible on a single line but the TTR extractor doesn't support line breaks yet.
```
Comment (akien-mga) [inline editor/animation_track_editor.cpp:6115]: Did you check that this comment was indeed no longer relevant?

Comment (nathanfranke) [reply editor/animation_track_editor.cpp:6115]: I assumed all of TTR was through the macro, which should work with this string concatenation. Is there an external program that parses these? If so, should it be in GitHub workflows?

Comment (akien-mga) [reply editor/animation_track_editor.cpp:6115]: See `editor/translations/extract.py`. The script parses the codebase to extract translatable strings, and I don't think the regex handles multilines.

Comment (akien-mga) [reply editor/animation_track_editor.cpp:6115]: Yep tested this on the `3.x` branch (which the current translations are synced with) and that breaks it:
```diff
diff --git a/editor/animation_track_editor.cpp b/editor/animation_track_editor.cpp
index ea7c6763ca..b63c28c58b 100644
--- a/editor/animation_track_editor.cpp
+++ b/editor/animation_track_editor.cpp
@@ -5750,7 +5750,9 @@ float AnimationTrackEditor::snap_time(float p_value, bool p_relative) {
 
 void AnimationTrackEditor::_show_imported_anim_warning() const {
 	// It looks terrible on a single line but the TTR extractor doesn't support line breaks yet.
-	EditorNode::get_singleton()->show_warning(TTR("This animation belongs to an imported scene, so changes to imported tracks will not be saved.\n\nTo enable the ability to add custom tracks, navigate to the scene's import settings and set\n\"Animation > Storage\" to \"Files\", enable \"Animation > Keep Custom Tracks\", then re-import.\nAlternatively, use an import preset that imports animations to separate files."),
+	EditorNode::get_singleton()->show_warning(TTR("This animation belongs to an imported scene, "
+			"so changes to imported tracks will not be saved.\n\n"
+			"To enable the ability to add custom tracks, navigate to the scene's import settings and set\n\"Animation > Storage\" to \"Files\", enable \"Animation > Keep Custom Tracks\", then re-import.\nAlternatively, use an import preset that imports animations to separate files."),
 			TTR("Warning: Editing imported animation"));
 }
 
diff --git a/editor/translations/editor.pot b/editor/translations/editor.pot
index 9be74af0de..7f609e1f7b 100644
--- a/editor/translations/editor.pot
+++ b/editor/translations/editor.pot
@@ -1585,19 +1585,6 @@ msgstr ""
 msgid "Anim Add RESET Keys"
 msgstr ""
 
-#: editor/animation_track_editor.cpp
-msgid ""
-"This animation belongs to an imported scene, so changes to imported tracks "
-"will not be saved.\n"
-"\n"
-"To enable the ability to add custom tracks, navigate to the scene's import "
-"settings and set\n"
-"\"Animation > Storage\" to \"Files\", enable \"Animation > Keep Custom "
-"Tracks\", then re-import.\n"
-"Alternatively, use an import preset that imports animations to separate "
-"files."
-msgstr ""
-
 #: editor/animation_track_editor.cpp
 msgid "Warning: Editing imported animation"
 msgstr ""
```

Comment (nathanfranke) [reply editor/animation_track_editor.cpp:6115]: I updated this to have only one line. Good catch! I also created an issue to possibly improve this in the future: https://github.com/godotengine/godot/issues/63410

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**